gallery/plot_video_api.py

```diff
@@ -0,0 +1,333 @@
+"""
+=======================
+Video API
+=======================
+
+This example illustrates some of the APIs that torchvision offers for
+videos, together with the examples on how to build datasets and more.
+"""
+
+####################################
+# 1. Introduction: building a new video object and examining the properties
+# -------------------------------------------------------------------------
+# First we select a video to test the object out. For the sake of argument
+# we're using one from Kinetics400 dataset.
+# To create it, we need to define the path and the stream we want to use.
+
+import torch
+import torchvision
+
+# Download the sample video
+from torchvision.datasets.utils import download_url
+download_url(
+    "https://github.com/pytorch/vision/blob/master/test/assets/videos/WUzgd7C1pWA.mp4?raw=true",
+    ".",
+    "WUzgd7C1pWA.mp4"
+)
+
+"""
+Chosen video statistics:
+WUzgd7C1pWA.mp4
+  - source: kinetics-400
+  - video: H-264 - MPEG-4 AVC (part 10) (avc1)
+    - fps: 29.97
+  - audio: MPEG AAC audio (mp4a)
+    - sample rate: 48K Hz
+"""
+video_path = "./WUzgd7C1pWA.mp4"
+
+######################################
+# Streams are defined in a similar fashion as torch devices. We encode them as strings in a form
+# of `stream_type:stream_id` where `stream_type` is a string and `stream_id` a long int.
```

Comment:
Since we're dealing with rst and not markdown (sadly, this is what sphinx prefers), we'll need double backtics instead of single backticks. Could you also fix the remaining ones?